Bedre håndtering av nye varer i handlekurv

FredrikAugust opened the pull request:
Closes #34.

Du trenger nå ikke lenger akseptere hver enkelt vare, det er et litt enklere system for å legge til nye.

<img width="808" alt="Screenshot 2020-03-18 at 22 02 13" src="https://user-images.githubusercontent.com/7691547/77007224-219ffa80-6964-11ea-868c-c521bb4cf071.png">

Commit:
Legg til kart som viser alle ordrer

Commit:
Merge branch 'master' of github.com:s0lvang/karantenehjelpen into feature/map-markers

Commit:
Bedre håndtering av nye varer i handlekurven, og fikset addresse

Commit:
Merge branch 'master' of github.com:s0lvang/karantenehjelpen into feature/auto-newline

plusk commented:
åååååååååå digg

Commit:
Increase legibility on login button

FredrikAugust committed:
Merge branch 'master' into feature/auto-newline

s0lvang commented:
Hvis du legger til en vare så må den ha et navn for at du skal kunne slette den. Is a bit annoying

https://imgur.com/a/UryTDLH

FredrikAugust commented:
Er det der i min branch @s0lvang? Jeg gjorde så du ikke kan "låse inn" requests

FredrikAugust commented:
@s0lvang that's fixed in my branch

s0lvang commented:
uuups! pulled the branch, then followed to test on karantenehjelpen.no :sweat_smile:

FredrikAugust committed:
Fiks sletting av varer i handlekurv

Og bruk en key som ikke er basert kun på index. Er hvertfall bedre i React, så antar det er samme i vju

FredrikAugust commented:
PR er ferdig nå. Klar til å merges fra min side 🎉

s0lvang reviewed (approved):
Looks amazing <3

FredrikAugust committed:
Revert "Legg til kart som viser alle ordrer"

This reverts commit 1bca47c752829a0fa3b11a1b25dbe69acb18dc96.
